riscv/processor.cc

```diff
@@ -465,13 +466,15 @@ void state_t::reset(processor_t* const proc, reg_t max_isa)
     const reg_t senvcfg_mask = (proc->extension_enabled(EXT_ZICBOM) ? SENVCFG_CBCFE | SENVCFG_CBIE : 0) |
                               (proc->extension_enabled(EXT_ZICBOZ) ? SENVCFG_CBZE : 0) |
                               (proc->extension_enabled(EXT_ZICFILP) ? SENVCFG_LPE : 0);
+                              (proc->extension_enabled(EXT_ZICFISS) ? SENVCFG_SSE : 0);
     csrmap[CSR_SENVCFG] = senvcfg = std::make_shared<senvcfg_csr_t>(proc, CSR_SENVCFG, senvcfg_mask, 0);
     const reg_t henvcfg_mask = (proc->extension_enabled(EXT_ZICBOM) ? HENVCFG_CBCFE | HENVCFG_CBIE : 0) |
                               (proc->extension_enabled(EXT_ZICBOZ) ? HENVCFG_CBZE : 0) |
                               (proc->extension_enabled(EXT_SVADU) ? HENVCFG_ADUE: 0) |
                               (proc->extension_enabled(EXT_SVPBMT) ? HENVCFG_PBMTE : 0) |
                               (proc->extension_enabled(EXT_SSTC) ? HENVCFG_STCE : 0) |
                               (proc->extension_enabled(EXT_ZICFILP) ? HENVCFG_LPE : 0);
+                              (proc->extension_enabled(EXT_ZICFISS) ? HENVCFG_SSE : 0);
```

Comment:
Same here.

Reply:
Thanks, 
I am so sorry for my missing check.